config-zos.md

```diff
@@ -1,20 +1,55 @@
-_This document is a work in progress._
-
 # <a name="ZOSContainerConfiguration" />z/OS Container Configuration
 
 This document describes the schema for the [z/OS-specific section](config.md#platform-specific-configuration) of the [container configuration](config.md).
+The z/OS container specification uses z/OS UNIX kernel features like namespaces and filesystem jails to fulfill the spec.
+
+Applications expecting a z/OS environment will very likely expect these file paths to be set up correctly.
+
+The following filesystems SHOULD be made available in each container's filesystem:
+
+| Path     | Type   |
+| -------- | ------ |
+| /proc    | [proc][] |
+
+## <a name="configZOSNamespaces" />Namespaces
+
+A namespace wraps a global system resource in an abstraction that makes it appear to the processes within the namespace that they have their own isolated instance of the global resource.
+Changes to the global resource are visible to other processes that are members of the namespace, but are invisible to other processes.
+
+Namespaces are specified as an array of entries inside the `namespaces` root field.
+The following parameters can be specified to set up namespaces:
 
-## <a name="configZOSDevices" />Devices
+* **`type`** *(string, REQUIRED)* - namespace type. The following namespace types SHOULD be supported:
+    * **`pid`** processes inside the container will only be able to see other processes inside the same container or inside the same pid namespace.
+    * **`mount`** the container will have an isolated mount table.
```

Comment:
Is there a public man page like https://man7.org/linux/man-pages/man7/mount_namespaces.7.html ?